app/src/main/java/edu/uc/hornunjb/weatherwear/dto/Weather.kt

```diff
@@ -40,48 +16,12 @@ class Weather {
     var icon: String? = null
 }
 
-class Clouds {
-    @SerializedName("all")
-    var all: Float = 0.toFloat()
-}
 
-class Rain {
-    @SerializedName("3h")
-    var h3: Float = 0.toFloat()
-}
```

Comment:
I thought it would be better practice to put this class in its own file. What I found was it is better for classes to be alone.